content/en/user-manual/scripting/custom_engine.md

```diff
@@ -48,6 +48,11 @@ To launch with the minified build, use:
 
     https://launch.playcanvas.com/<scene_id>?use_local_engine=https://code.playcanvas.com/playcanvas-0.225.0.min.js
 
+<div class="alert alert-info">
+    The Editor only officially supports the current Engine release and the previous minor version. While it is sometimes possible to use an older version of the Engine, it is not a long-term supported workflow. We strongly recommend keeping live projects updated with the current Engine release.
+</div>
+
+
```

Comment:
Only put a single line between blocks of Markdown text. I should create a style guide. But you'll see it's like that throughout the entire content base.